demo/src/routes/menu/Versions.svelte

```diff
@@ -0,0 +1,86 @@
+<script lang="ts">
+	import {get, writable} from '@amadeus-it-group/tansu';
+	import type {ReadableSignal, Writable} from '@amadeus-it-group/tansu';
+	import type {Directive} from '@agnos-ui/svelte/types';
+	import {createHasFocus} from '@agnos-ui/svelte/services/focustrack';
+	import {onMount} from 'svelte';
+	import {page} from '$app/stores';
+	import {pathToRoot$, selectedFramework$} from '$lib/stores';
+
+	interface Version {
+		folder: string;
+		version: string;
+	}
+
+	const versionsParsed: Version[] = [{folder: 'latest', version: 'Latest'}];
+
+	onMount(() => {
+		fetch('https://amadeusitgroup.github.io/AgnosUI/versions.json').then((res) => {
```

Comment:
was thinking to add the pathToRoot/versions.json but it won't work on local (kinda useless but still interesting to have). WDYT?